### DISCUSSION TITLE
Create Issued Document - Acceptable values for "country" field

### DISCUSSION THREAD

**Discussion by Simonaester** [fattureincloud/api#20] [Q&A] [answered]
Buongiorno, ho notato un "comportamento" anomalo nella validazione del field entity->country nella chiamata  "create_issued_document".
Secondo la doc dovrebbe accettare un campo stringa, ma inserendo il nome sottoforma di codice paese (ad es: 'IT') oppure in inglese (es: 'Italy'), le api  generano un errore di validazione :

error":{"message":"Invalid ' 'request.","validation_result":{"data.entity.country":["The ' 'data.entity.country field is not valid."]}}}',
'code': 422, 'description': 'Unprocessable Entity'.

Qualcuno saprebbe aiutarmi?

**Comment by valmoz** [comment, marked as answer]
Ciao @Simonaester,
puoi ottenere i valori validi per il campo "country" utilizzando la chiamata [List Countries](https://developers.fattureincloud.it/api-reference/#get-/info/countries).
Il valore per l'Italia ad esempio dovrebbe essere "Italia". Lo so, non la scelta migliore possibile...

A disposizione per ulteriori informazioni.
Mauro

N.B.: di solito cerchiamo di usare l'inglese all'interno della community, per renderla fruibile alla platea più ampia possibile.
